packages/contract-wrappers/src/index.ts

```diff
@@ -11,50 +48,61 @@ export {
     ExchangeCancelUpToEventArgs,
     ExchangeAssetProxyRegisteredEventArgs,
     ExchangeContract,
-    DevUtilsContract,
-    ForwarderContract,
-    CoordinatorContract,
-    CoordinatorRegistryEventArgs,
-    CoordinatorRegistryEvents,
-    CoordinatorRegistryCoordinatorEndpointSetEventArgs,
-    CoordinatorRegistryContract,
-    IValidatorContract,
-    IWalletContract,
+    ExchangeProtocolFeeCollectorAddressEventArgs,
+    ExchangeProtocolFeeMultiplierEventArgs,
+    ExchangeTransactionExecutionEventArgs,
+} from './generated-wrappers/exchange';
+export { ForwarderContract } from './generated-wrappers/forwarder';
+export { IValidatorContract } from './generated-wrappers/i_validator';
+export { IWalletContract } from './generated-wrappers/i_wallet';
+export { OrderValidatorContract } from './generated-wrappers/order_validator';
+export {
+    StakingAuthorizedAddressAddedEventArgs,
+    StakingAuthorizedAddressRemovedEventArgs,
+    StakingContract,
+    StakingEpochEndedEventArgs,
+    StakingEpochFinalizedEventArgs,
+    StakingEventArgs,
+    StakingEvents,
+    StakingExchangeAddedEventArgs,
+    StakingExchangeRemovedEventArgs,
+    StakingMakerStakingPoolSetEventArgs,
+    StakingMoveStakeEventArgs,
+    StakingOperatorShareDecreasedEventArgs,
+    StakingOwnershipTransferredEventArgs,
+    StakingParamsSetEventArgs,
+    StakingRewardsPaidEventArgs,
+    StakingStakeEventArgs,
+    StakingStakingPoolCreatedEventArgs,
+    StakingStakingPoolEarnedRewardsInEpochEventArgs,
+    StakingUnstakeEventArgs,
+} from './generated-wrappers/staking';
+export {
+    StakingProxyAuthorizedAddressAddedEventArgs,
+    StakingProxyAuthorizedAddressRemovedEventArgs,
+    StakingProxyContract,
+    StakingProxyEventArgs,
+    StakingProxyEvents,
+    StakingProxyOwnershipTransferredEventArgs,
+    StakingProxyStakingContractAttachedToProxyEventArgs,
+    StakingProxyStakingContractDetachedFromProxyEventArgs,
+} from './generated-wrappers/staking_proxy';
```

Comment:
Feeling similarly about staking contracts. Thoughts?

Reply:
@dekz weighed in and let me know Staking/StakingProxy are needed!